lib/viewmodel/club_detail/club_detail_viewmodel.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter_template/model/webservice/club/club.dart';
+import 'package:flutter_template/navigator/mixin/error_navigator.dart';
+import 'package:flutter_template/repository/club_detail/club_detail_repository.dart';
+import 'package:icapps_architecture/icapps_architecture.dart';
+import 'package:injectable/injectable.dart';
+
+@injectable
+class ClubDetailViewModel with ChangeNotifierEx {
```

Comment:
In this pull request I would already pass the ID to this club detail screen. That way this screen is fully finished. And the clubs viewmodel. Will contain that specific ID when opening 1 clubdetail.